docker/app/customizations.php.ini

```diff
@@ -0,0 +1,3 @@
+memory_limit = 256M
+post_max_size = 60M
+upload_max_filesize = 60M
```

Comment:
this is awesome - well done in pulling these customizations out!

Reply:
thanks, I was happy to learn more about php's config options, this is much better.